experimental/traffic-portal/src/app/core/certs/cert-viewer/cert-viewer.component.html

```diff
@@ -0,0 +1,77 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card appearance="outlined" class="page-content">
+	<tp-loading *ngIf="dsCert === undefined"></tp-loading>
+	<mat-card-content *ngIf="dsCert !== undefined">
+		<mat-tab-group #matTab>
+			<mat-tab label="Input Certificate" *ngIf="!dsCert">
+					<form ngNativeValidate (submit)="process(true)" class="tab-content">
+						<mat-form-field>
+							<mat-label>Certificate Chain</mat-label>
+							<textarea matInput name="custom" [(ngModel)]="inputCert" cdkAutosizeMaxRows="15" cdkTextareaAutosize required></textarea>
+						</mat-form-field>
+						<button mat-raised-button type="submit">Submit</button>
+					</form>
+			</mat-tab>
+			<mat-tab label="Certificate Details" [disabled]="certChain.length === 0">
+				<div class="tab-content">
+					<mat-form-field>
+						<mat-label>Detected Order</mat-label>
+						<input matInput readonly name="order" [(ngModel)]="certOrder"/>
```

Comment:
"text" is the default if not specified. It's there in a lot of other cases because I like to put it down explicitly, but the linter doesn't require it so I would let it go.